packages/vite/src/node/cli.ts

```diff
@@ -133,7 +135,7 @@ cli
   .option('-w, --watch', `[boolean] rebuilds when modules have changed on disk`)
   .action(async (root: string, options: BuildOptions & GlobalCLIOptions) => {
     const { build } = await import('./build')
-    const buildOptions = cleanOptions(options) as BuildOptions
+    const buildOptions = cleanOptions(options)
```

Comment:
We should add the explicit type here, to always get safe that it doesn't pass e.g. to few properties

```suggestion
    const buildOptions: BuildOptions = cleanOptions(options)
```

Reply:
Fix it.